docker/importer/importer.py

```diff
@@ -136,8 +145,36 @@ def import_new_oss_fuzz_entries(self, repo, oss_fuzz_source):
       return
 
     logging.info('Commiting and pushing new entries')
-    osv.push_source_changes(repo, 'Import from OSS-Fuzz',
-                            self._git_callbacks(oss_fuzz_source))
+    if osv.push_source_changes(repo, 'Import from OSS-Fuzz',
+                               self._git_callbacks(oss_fuzz_source)):
+      ndb.put_multi(exported)
+
+  def schedule_regular_updates(self, repo, oss_fuzz_source):
+    """Schedule regular OSS-Fuzz updates."""
+    for bug in osv.Bug.query(osv.Bug.status == osv.BugStatus.PROCESSED,
+                             osv.Bug.fixed == ''):
+      if bug.source_of_truth == osv.SourceOfTruth.SOURCE_REPO:
```

Comment:
Helper function for this repeat ?

if bug.source_of_truth == osv.SourceOfTruth.SOURCE_REPO:
        self._request_analysis(oss_fuzz_source, repo, osv.source_path(bug))
      else:
        self._request_internal_analysis(bug)

Reply:
Done!